src/main/java/seedu/address/model/person/Person.java

```diff
@@ -25,18 +25,21 @@
     // Data fields
     private final Address address;
     private final Optional<Property> property;
+    private final Optional<Property> preference;
```

Comment:
Is preference supposed to represent a specific `Property` that the client wants to buy or the general criteria of the `Property` that needs to be satisfied?